node-optional.el8.repo

```diff
@@ -6,12 +6,58 @@ mirrorlist = http://mirrorlist.centos.org/?release=$stream&arch=$basearch&repo=B
 gpgcheck = 1
 enabled = 1
 gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial
-includepkgs = dnf-plugin-subscription-manager fcoe-utils python3-librepo subscription-manager usermode
+includepkgs =
```

Comment:
nit: can we sort this alphabetically?

Reply:
I'd personally actually do the opposite: Order in order of addition, and add comments (if possible) - e.g. "# ipa dependencies".

Reply:
I'll try making it grouped by reason and sorted alphabetically within the group.